src/Component/BlogPage.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import './BlogPage.css'
+
+import BlogMiddleComp from './BlogMiddleComp'
+import { data } from '../data'
+function BlogPage() {
+
+    const [searchTerm, setSearchTerm] = useState('');
+    const handleSearch = (e) => {
+        setSearchTerm(e.target.value)
+        console.log(searchTerm)
```

Comment:
Console log should be removed. This function wasn't really required. Could have been done inline.:

```
(e) => setSearchTerm(e.target.value)
```